CardsAgainst/Views/TouchableLabel.swift

```diff
@@ -11,32 +11,37 @@ import UIKit
 final class TouchableLabel: UILabel {
     var placeholderRanges = [NSRange]()
 
-    override init() {
+    init() {
         super.init(frame: CGRectZero)
         userInteractionEnabled = true
     }
+    
+    override init(frame: CGRect) {
+        super.init(frame:frame)
+        userInteractionEnabled = true
+    }
 
-    required init(coder aDecoder: NSCoder) {
+    required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 
     private func setLastTokenAlpha(alpha: CGFloat) {
         if let lastRange = placeholderRanges.last {
-            let mAttributedText = attributedText.mutableCopy() as NSMutableAttributedString
+            let mAttributedText = attributedText!.mutableCopy() as! NSMutableAttributedString
             mAttributedText.addAttribute(NSForegroundColorAttributeName, value: tintColor.colorWithAlphaComponent(alpha), range: lastRange)
             attributedText = mAttributedText
         }
     }
-
-    override func touchesBegan(touches: NSSet, withEvent event: UIEvent) {
+    
```

Comment:
there's a lot of this added whitespace on empty lines. could you please revert that?